Applications/Spire/Include/Spire/UiViewer/StandardUiProfiles.hpp

```diff
@@ -188,6 +188,9 @@ namespace Spire {
   /** Returns a UiProfile for the PercentBox. */
   UiProfile make_percent_box_profile();
 
+  /** Returns a UiProfile for the PlaybackSpeedBox. */
+  UiProfile make_playback_speed_box_profile();
+
```

Comment:
Don't add a UI Viewer entry for this, otherwise the UI Viewer will get bloated with a bunch of trivial components.

UI Viewer is appropriate when you need to test and verify components that implement new functionality, not for components that are instances of existing components.